src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,7 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
Casting the result of getInstance() to ProductService is not safe. If the injector fails to provide an instance of ProductService, this will result in a ClassCastException. It would be better to have getInstance return a typed object or handle the potential exception here.